std_msgs/Time stamp # (Two-integer timestamp that is expressed as seconds and nanoseconds)
string id
bool in_current_action
bool enabled
float64[] internal_activation_function
float64[] external_activation_function
float64[] reference_rate

